benchexec/tablegenerator/test_integration/__init__.py

```diff
@@ -165,6 +164,12 @@ def expected_diff_file_name(ending):
         if expected_counts:
             # output of table-generator should end with statistics about regressions
             counts = output[output.find("REGRESSIONS") :].strip()
+            # Add \r before all \n on Windows if it's not there already
+            if platform.system() == "Windows":
+                line_feed_without_carriage = re.compile("(?<!\\r)\\n")
+                expected_counts = re.sub(
+                    line_feed_without_carriage, "\\r\\n", expected_counts
+                )
```

Comment:
Would it be possible to simplify this to something like `counts = counts.replace('\r\n', '\n')`?

I would suggest to normalize the actual output to `\n` (more common) and we can do this unconditionally, no need for a platform check.

Reply:
Basically call the utility function that you already have, just simplify its content and rename it to `normalize_line_endings`.

Reply:
Yes, replacing the line endings in this direction actually seems to make more sense and is simpler. I've applied the changes.

Should we also do the same for the `fix_path_if_on_windows` function, i.e. always call the `replace` function regardless of the current OS? No problems occured when I tried it out. Would that make sense?

Reply:
`\` is an allowed file-name character on Linux, so here we should not do this.